sdk/textanalytics/Azure.AI.TextAnalytics/tests/RecognizeHealthcareEntitiesTests.cs

```diff
@@ -0,0 +1,230 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using NUnit.Framework;
+
+namespace Azure.AI.TextAnalytics.Tests
+{
+    public class RecognizeHealthcareEntitiesTests : TextAnalyticsClientLiveTestBase
+    {
+        public RecognizeHealthcareEntitiesTests(bool isAsync) : base(isAsync) { }
+
+        private const string singleEnglish = "Subject is taking 100mg of ibuprofen twice daily";
+
+        private static List<string> batchConvenienceDocuments = new List<string>
+        {
+            "Subject is taking 100mg of ibuprofen twice daily",
+            "Can cause rapid or irregular heartbeat, delirium, panic, psychosis, and heart failure."
+        };
+
+        private static List<TextDocumentInput> batchDocuments = new List<TextDocumentInput>
+        {
+            new TextDocumentInput("1", "Subject is taking 100mg of ibuprofen twice daily")
+            {
+                 Language = "en",
+            },
+            new TextDocumentInput("2", "Can cause rapid or irregular heartbeat, delirium, panic, psychosis, and heart failure.")
+            {
+                 Language = "en",
+            }
+        };
+
+        [Test]
+        public async Task RecognizeHealthcareEntitiesTest()
+        {
+            TextAnalyticsClient client = GetClient();
+            string document = singleEnglish;
+
+            HealthcareOperation operation = await client.StartHealthcareAsync(document);
+
+            await operation.WaitForCompletionAsync();
+
+            RecognizeHealthcareEntitiesResultCollection resultCollection = operation.Value;
+
+            Assert.AreEqual(1, resultCollection.Count);
```

Comment:
U could also do
`DocumentHealthcareResult result = resultCollection.Single();`

It will fail if the collection has more than one item and it will give u the item so no need to do a for loop